core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -12942,4 +12943,47 @@ private String toSqlWithBloat(RelNode root, int bloat) {
     assertThat(toSql(root, DatabaseProduct.CALCITE.getDialect()), isLinux(expectedSql));
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBiqQuery));
   }
+
+  @Test public void testJSON_OBJECT() {
```

Comment:
make it testJsonObjectFunction

Reply:
changed testJSON_OBJECT to testJsonObjectFunction